UEBPrep/Gemfile.lock

```diff
@@ -42,7 +42,6 @@ GEM
       timers (~> 4.0.0)
     childprocess (0.5.6)
       ffi (~> 1.0, >= 1.0.11)
-    client_side_validations (3.2.7)
     climate_control (0.0.3)
```

Comment:
Mistake on my part here. You should always commit your gemfile and gemfile.lock changes in one commit.